doc/nrf/ug_tfm.rst

```diff
@@ -17,15 +17,14 @@ TF-M is the reference implementation of `Platform Security Architecture (PSA)`_.
 It provides a highly configurable set of software components to create a Trusted Execution Environment.
```

Comment:
Where does it make most sense to add link to https://armmbed.github.io/mbed-crypto/html/? This paragraph? Or with the sentence above it?

Reply:
Added under section 'Enabling secure services'.